### PR TITLE
MC-17689: added api doc for flagging invoices

### PR DESCRIPTION
**Pull request by jasondias9**
#### Changes made

- added doc for flagging an invoice

#### Related PRs
- https://github.com/cloudops/cloudmc-core/pull/1970

### CODE REVIEW

File: source/includes/administration/_invoices.md [hunk 1]
```diff
@@ -2136,3 +2136,52 @@ curl  -X GET 'https://cloudmc_endpoint/rest/invoices/download?invoice_id=3f7b7cc
...
+```shell
+# Approve a draft invoice
+curl -X POST "https://cloudmc_endpoint/rest/invoices/20e9b8d8-b1cb-4462-b6e8-fbb8416b2cbb/flag" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+| Required | &nbsp; |
+| --- | --- |
+| `invoiceId`<br/>*UUID* | The id of the invoice to be flagged |
+
```
Comment (dpmurray21) [inline source/includes/administration/_invoices.md:2161]: should you also add `message` here?

**Review by dpmurray21 — CHANGES_REQUESTED** [review]

File: source/includes/administration/_invoices.md [hunk 2]
```diff
@@ -2136,3 +2136,52 @@ curl  -X GET 'https://cloudmc_endpoint/rest/invoices/download?invoice_id=3f7b7cc
...
 | --- | --- |
 | `invoice_id`<br/>*UUID* | The id of the invoice. |
+
+
+<!-------------------- FLAG INVOICE -------------------->
+
+`POST /invoices/:invoiceId/flag`
+
+Flag an invoice with a message to prevent the invoice from being automatically issued to the customer. 
+Once an invoice is flagged, pricing configuration changes can be made.
```
Comment (m-orfanos) [inline source/includes/administration/_invoices.md]: the wording implies changes can't be made unless it's flagged imo

**Review by m-orfanos — APPROVED** [review]

File: source/includes/administration/_invoices.md (hunk 2 above)
Comment (jasondias9) [reply source/includes/administration/_invoices.md]: [`c9b3740` (#501)](https://github.com/cloudops/cloudmc-api-docs/pull/501/commits/c9b3740770f3467a99dcb14babb00eb5cc8eda90)

File: source/includes/administration/_invoices.md (hunk 1 above)
Comment (jasondias9) [reply source/includes/administration/_invoices.md:2161]: [`c9b3740` (#501)](https://github.com/cloudops/cloudmc-api-docs/pull/501/commits/c9b3740770f3467a99dcb14babb00eb5cc8eda90)

**Review by dpmurray21 — APPROVED** [review]

**Pull request merged**